detectron2/data/transforms/transform_gen.py

```diff
@@ -113,18 +114,16 @@ def __repr__(self):
 
 class RandomFlip(TransformGen):
     """
-    Flip the image horizontally with the given probability.
-
-    TODO Vertical flip to be implemented.
+    Flip the image horizontally or vertically with the given probability.
     """
 
-    def __init__(self, prob=0.5):
+    def __init__(self, prob=0.5, horiz=True, vert=False):
```

Comment:
Could you use the signature
```
__init__(self, prob=0.5, *, horizontal=True, vertical=False):
```
They now becomes an API so we expect clearer names. Also it's good to prevent users writing `RandomFlip(0.5, True)` since it's generally confusing.

Reply:
Done.